app/assets/javascripts/discourse/components/topic-navigation.js.es6

```diff
@@ -109,6 +111,55 @@ export default Ember.Component.extend({
     }
   },
 
+  _panOpenClose(offset, velocity, direction) {
```

Comment:
another nitpick here, but not a huge fan of `_panOpenClose` name here. Could we have:

```
_panOpen(offset, velocity, direction) {
  this._panToggle(offset, velocity, "open");
},

_panClose(offset, velocity, direction) {
  this._panToggle(offset, velocity, "close");
},

_panToggle(offset, velocity, direction) {
}
```